src/gateways/createPracticeQuestionsLoader.ts

```diff
@@ -0,0 +1,15 @@
+import { PracticeQuestionsSummary } from '../app/content/practiceQuestions/types';
+import mockPhysics from '../test/fixtures/practice/cce64fde-f448-43b8-ae88-27705cceb0da.json';
```

Comment:
i wanted to import this with a lazy import using a variable path based on the uuid passed in. then even if the data is not production ready the code will be. the lazy import means we're not bundling all the data in the default js file, but will load only the necessary books data we need after page load. 

also my intent wasn't to have this be a test file, but to put it in something like `rex-web/data/practice/summary/cce64fde-f448-43b8-ae88-27705cceb0da.json` we will use the same wiring in the loader when we go to production, we'll just have to add additional build logic to populate the files

Reply:
> i wanted to import this with a lazy import using a variable path based on the uuid passed in. then even if the data is not production ready the code will be. the lazy import means we're not bundling all the data in the default js file, but will load only the necessary books data we need after page load.

By "lazy loading" do you mean `React.lazy(() => import('.../bookUid.json'))`? If yes then I think it is not possible to pass `bookUid` dynamically because webpack have to know from the beginning what is the name of the file to handle it properly

We could, however, add `/data/practice/summary/cce64fde-f448-43b8-ae88-27705cceb0da.json` to the `static` folder that holds other assets and get it's content with `fetch`. Is this acceptable?

Reply:
i want you to use the 
```
const data = import(`/data/practice/summary${uuid}.json`);
```
syntax so that the data files will not be bundled with the rest of the js but will be loaded as needed